biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblId.java

```diff
@@ -0,0 +1,108 @@
+package org.biojava.nbio.core.sequence.io.embl;
```

Comment:
All source code in biojava requires the license header

Reply:
done.